packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -344,6 +344,35 @@
           "dark": "resources/dark/stopWatching.svg"
         }
       }
-    ]
+    ],
+    "configuration": {
+      "type": "object",
+      "title": "%force_lightning_lwc_preferences%",
+      "properties": {
+        "salesforcedx-vscode-lwc.rememberDevice": {
```

Comment:
Add some hierarchy here:
For example: "salesforcedx-vscode-lwc.preview.preview.rememberDevice"

Reference:
https://code.visualstudio.com/api/references/contribution-points#Configuration-schema
> Entries will be grouped according to the hierarchy established in your keys.